observer.go

```diff
@@ -22,12 +22,16 @@ package jaeger
 
 import opentracing "github.com/opentracing/opentracing-go"
 
-// Observer can be registered with the Tracer to receive notifications about new Spans.
+// Observer can be registered with the Tracer to receive notifications about
+// new Spans. Note that this interface is now Deprecated, see:
```

Comment:
See the recommended format for deprecation notice https://rakyll.org/deprecated/

```
[blank line]
Deprecated: use jaeger.ContribObserver instead.
```